next.config.mjs

```diff
@@ -1,4 +1,15 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
-
-export default nextConfig;
+import { i18n } from './next-i18next.config.js';
+const nextConfig = {
+    async rewrites() {
+      return [
+        {
+          source: '/api/:path*',
+          destination: 'http://127.0.0.1:3000/api/:path*', 
```

Comment:
Is there any ways to avoid hardcode the path and port like "127.0.0.1:3000" ?

Reply:
First, we should never hard-code values in the code. Secondly, our Next.js framework only handles front-end routing, while the backend is managed by the Express framework, so there is no need for redirection configuration.